packages/teleport/src/Databases/AddDialog/AddDialog.tsx

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2021 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import Dialog, {
+  DialogHeader,
+  DialogTitle,
+  DialogContent,
+  DialogFooter,
+} from 'design/Dialog';
+import { Text, Box, ButtonSecondary, Link } from 'design';
+import Select, { Option } from 'shared/components/Select';
+import * as links from 'teleport/services/links';
+import { AuthType } from 'teleport/services/user';
+import { DbType, DbProtocol } from 'teleport/services/databases';
+import {
+  formatDatabaseInfo,
+  DatabaseInfo,
+} from 'teleport/services/databases/makeDatabase';
+import TextSelectCopy from 'teleport/components/TextSelectCopy';
+
+export default function AddDialog({ user, version, authType, onClose }: Props) {
```

Comment:
Lets rename it to `AddDatabase`. It's easier to search and less generic than "AddDialog". Another benefit is the name is implementation agnostic meaning that if we decide to use something else (not the dialog) we wont need to change the component name.